### PR TITLE
Feature/multi tenant api

### PR DESCRIPTION
**Pull request by rfliam**

### CODE REVIEW

File: src/test/java/com/bitmovin/api/examples/MultiTenantEncoding.java [hunk 1]
```diff
@@ -0,0 +1,343 @@
...
+import org.junit.Test;
+
+import java.io.IOException;
+import java.net.URISyntaxException;
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.Date;
+
+/**
+ * Created by Roland Kersche on 01.06.17.
```
Comment (jamescyeh) [inline src/test/java/com/bitmovin/api/examples/MultiTenantEncoding.java]: Ahem

**Review by jamescyeh — APPROVED** [review]
One minor change, otherwise LGTM.

File: src/main/java/com/bitmovin/api/BitmovinApi.java [hunk 2]
```diff
@@ -158,6 +166,9 @@ private void setDefaultHeaders() throws IOException
     {
         this.defaultHeaders.put("Content-Type", "application/json");
         this.defaultHeaders.put("X-Api-Key", this.apiKey);
+        if (this.tenantID != null) {
```
Comment (dominic-miglar) [inline src/main/java/com/bitmovin/api/BitmovinApi.java]: code style -> { goes to next line.

File: src/test/java/com/bitmovin/api/examples/MultiTenantEncoding.java [hunk 3]
```diff
@@ -0,0 +1,343 @@
...
+
+    private AudioAdaptationSet addAudioAdaptationSetToPeriodWithRoles(DashManifest manifest, Period period, String lang) throws URISyntaxException, BitmovinApiException, RestException, UnirestException, IOException
+    {
+        AudioAdaptationSet a = new AudioAdaptationSet();
+        a.setLang(lang);
+        a = bitmovinApi.manifest.dash.addAudioAdaptationSetToPeriod(manifest, period, a);
+        return a;
+    }
+
+    private AudioAdaptationSet addAudioAdaptationSet(DashManifest manifest, Period period, AudioAdaptationSet audioAdaptationSet)
```
Comment (dominic-miglar) [inline src/test/java/com/bitmovin/api/examples/MultiTenantEncoding.java]: is this method even used?

**Review by dominic-miglar — CHANGES_REQUESTED** [review]

**Comment by rfliam** [comment]
Changes made re-feedback.  Please re-review asap @dominic-miglar @jamescyeh .

**Comment by rfliam** [comment]
Ping. @dominic-miglar

**Review by dominic-miglar — APPROVED** [review]

File: README.md [hunk 4]
```diff
@@ -11,7 +11,7 @@ Add this to your pom.xml:
 <dependency>
       <groupId>com.bitmovin.api</groupId>
       <artifactId>bitmovin-java</artifactId>
-      <version>1.36.0</version>
+      <version>1.37.0</version>
```
Comment (rkersche) [inline README.md]: That will be changed during a release

File: pom.xml [hunk 5]
```diff
@@ -4,7 +4,7 @@
     <modelVersion>4.0.0</modelVersion>
     <groupId>com.bitmovin.api</groupId>
     <artifactId>bitmovin-java</artifactId>
-    <version>1.36.0</version>
+    <version>1.37.0</version>
```
Comment (rkersche) [inline pom.xml]: That will be changed during a release

File: src/main/java/com/bitmovin/api/BitmovinApi.java [hunk 6]
```diff
@@ -47,12 +47,13 @@
     private static String X_API_CLIENT_VERSION_CONFIGURATION_KEY = "x-api-client-version";
 
     private static String DEFAULT_X_API_CLIENT = "bitmovin-java-api";
-    private static String DEFAULT_X_API_CLIENT_VERSION = "1.36.0";
+    private static String DEFAULT_X_API_CLIENT_VERSION = "1.37.0";
```
Comment (rkersche) [inline src/main/java/com/bitmovin/api/BitmovinApi.java]: That will be changed during a release

**Review by rkersche — CHANGES_REQUESTED** [review]

**Comment by rfliam** [comment]
This is still outstanding.

**Review by rkersche — APPROVED** [review]

**Pull request merged**